### DISCUSSION TITLE
Translatable badges in sidebar

### DISCUSSION THREAD

**Discussion by dragomano** [withastro/starlight#1721] [Feature Requests]
### What version of `starlight` are you using?

0.21.4

### What is your idea?

```js
{
	label: 'Resources',
	badge: 'New', // <= it should be translatable
	//...
},
```

### Why is this feature necessary?

For completeness of translation :)

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Agreed! Something we missed in the API design when adding this feature, but would be good to add. Maybe once we figure this out for the site title in #1620, we can then use a similar pattern for category badges like this.

**Comment by HiDeoo** [comment]
Starting with [Starlight 0.28.0](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.28.0), sidebar badges can now be [translated](https://starlight.astro.build/guides/sidebar/#internationalization-with-badges):

```js
{
  slug: 'constellations/andromeda',
  badge: {
    text: {
      en: 'New',
      'pt-BR': 'Novo',
    },
  },
}
```
